plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -463,7 +463,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             while (entries.hasMoreElements()) {
                 JarEntry entry = entries.nextElement();
                 String name = entry.getName();
-                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
+                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-(alpha|beta|RC|rc)-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
```

Comment:
qualifiers should be case insensitive...

untested. ->  `"WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:i-(alpha|beta|RC|rc)-[0-9.]+)?(?:i-SNAPSHOT)?)[.]jar"`

(is there a test case for this?)